data/campaigns/Legend_of_Wesmere/scenarios/chapter1/03_Kalian_under_Attack.cfg

```diff
@@ -180,7 +180,7 @@
         user_team_name= _ "Enemies"
         fog=yes
         shroud=yes
-        share_view=yes
+        share_vision=all
```

Comment:
Just a comment about the scenario, not an issue with the current PR.

It seems strange that there are settings for sharing vision between the AI sides. I can only assume that it's either a copy-and-paste from the human teams, or that there was an idea of having a human-playing-elves vs human-playing-orcs match.

Reply:
I noticed that as well, but I thought it was to do with how the AI operates. No idea if they are normally able to share information without this explicit indicator, though I imagine they probably do.

Reply:
Just found that the AI is more complex than I thought - some of the candidate actions do check if units are visible. I incorrectly thought it ignored fog of war completely.

No idea if that makes a different on this scenario.